morse_code.rb

```diff
@@ -0,0 +1,34 @@
+def decode_char(character)
```

Comment:
- [ ] The first test did not return the expected output. We are required to output `"A"` but we had this instead `:A`. Below is the screenshot in my testing environment
![Screenshot from 2022-10-27 13-42-56](https://user-images.githubusercontent.com/46388113/198301989-dedb7b7d-a30c-4a9f-8048-3345f8f43bae.png)

Reply:
Hello @addod19 
Thanks for the review. But I got some issues with the changes you required. 
When testing it using the ` print decode_char('.-')` and running the `ruby .\morse_code.rb` command on the terminal it gives us the required output which is `A`. So we don't really know how to address this issue

Reply:
Here is a screenshot of my result;
![pic](https://user-images.githubusercontent.com/98928638/198335518-1932052b-c30e-4b19-b37d-2ce5960b8730.PNG)